mne/viz/tests/test_topomap.py

```diff
@@ -109,6 +110,13 @@ def test_plot_topomap():
                     for x in subplot.get_children()
                     if isinstance(x, matplotlib.text.Text)))
 
+    # Plot array
+    for ch_type in ('mag', 'grad'):
+        evoked_ = evoked.copy().pick_types(eeg=False, meg=ch_type)
```

Comment:
Shouldn't need it, `seeg=False` by default. Actually you don't need the `eeg=False` either